tests/functional/modules/test_zos_copy_func.py

```diff
@@ -534,8 +534,8 @@ def test_copy_dir_to_existing_uss_dir_not_forced(ansible_zos_module):
         for result in copy_result.contacted.values():
             assert result.get("msg") is not None
             assert result.get("changed") is False
-            assert "Error" in result.get("msg")
-            assert "EDC5117I" in result.get("stdout")
+            assert "error" in result.get("msg").lower()
+            # assert "EDC5117I" in result.get("stdout")  # No longer valid, because different utility was used
```

Comment:
I find this interesting that `EDC5117I` will not be seen, its ok we don't get that specific but this causes concern to me because that error is a result of mkdir failing, see [here](https://www.ibm.com/docs/en/zos/2.2.0?topic=SSLTBW_2.2.0/com.ibm.zos.v2r2.ceea900/cs00223.html)

The test case name is test_copy_dir_to_existing_uss_dir_not_forced which implies it is going to copy to a existing directory without force and expected to fail, probably by only calling a mkdir() which is why we saw the error. Ideally i would have rather had a check done first to see if the directory exists before calling a mkdir which is what I can hypothesize without following the code path this test is intending to drive. 

I wonder what changed to not make the EDC5117I not appear.